kubernetes/deployment.template.yaml

```diff
@@ -0,0 +1,216 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ${SERVICE_NAME}
+  namespace: ${NAMESPACE}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ${SERVICE_NAME}
+  template:
+    metadata:
+      labels:
+        app: ${SERVICE_NAME}
+    spec:
+      initContainers:
+        - name: init-wait-db
+          image: busybox
+          command: ['sh', '-c', 'for i in $(seq 1 300); do nc -zvw1 shared-database 5432 && exit 0 || sleep 3; done; exit 1']
+      containers:
+        - image: ${IMAGE_NAME}
+          name: ${SERVICE_NAME}
+          envFrom:
+            - configMapRef:
+                name: ${SERVICE_NAME}-config
+          env:
+#            Import variables from vault
+            - name: S2S_KEY
+              value: '${S2S_TOKEN}'
+#            Import from main.tf
+#            This is to use the AAT database
+            - name: POSTGRES_HOST
+              value: "${POSTGRES_HOST}"
+            - name: POSTGRES_DATABASE
+              value: "${POSTGRES_DATABASE}"
+            - name: POSTGRES_USER
+              value: "${POSTGRES_USER}"
+            - name: POSTGRES_PASSWORD
+              value: "${POSTGRES_PASSWORD}"
+
+#            db
+            - name: SPRING_DATASOURCE_URL
+              value: 'jdbc:postgresql://shared-database:5432/emannotationapp?ssl=true'
+            - name: SPRING_DATASOURCE_USERNAME
+              value: 'emannotationapp'
+            - name: SPRING_DATASOURCE_PASSWORD
+              value: 'emannotationapp'
+            - name: ENABLE_DB_MIGRATE
+              value: 'true' #TODO: Alec set it as true, but main.tf has value = false. What should I set it to?
+
+#            TODO: IDAM_API_BASE_URI and S2S_BASE_URI not included, should they be? why/why not?
+#            in application.yaml, theyre http://localhost:4501 and 4502
+#            but in variables.tf they're http://betaDevBccidamAppLB.reform.hmcts.net:80 and http://rpe-service-auth-provider-preview.service.core-compute-preview.internal
+            - name: REFORM_SERVICE_NAME
+              value: 'em-anno' # See em_anno_app_url in variables.tf
+            - name: REFORM_TEAM
+              value: 'evidence'
+            - name: REFORM_SERVICE_TYPE
+              value: 'em-anno' #TODO: Alec set it to 'em-anno', but main.tf has 'java'. What should I set it to?
+            - name: REFORM_ENVIRONMENT
+              value: 'preview' # TODO Why is preview being used if we're replacing it with AKS?
```

Comment:
aks doesn't have a different env name, its still preview.
confusing and open to ideas for a different name if it would help (not here, on #aks-cnp)